data/presets/X-Pressive/X-Pressive1.xpf

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<!DOCTYPE lmms-project>
+<lmms-project version="1.0" creator="LMMS" creatorversion="1.2.0-rc2.6" type="instrumenttracksettings">
+  <head/>
+  <instrumenttracksettings muted="0" type="0" name="X-Pressive" solo="0">
```

Comment:
The user will want to have some good presets to look at for this one. In any case I don't think this should hold back a merge. You or someone else can tweak the names later and it's easier for people to try it out and make presets if it's merged anyway.